ext/semian/resource.c

```diff
@@ -234,6 +238,25 @@ check_tickets_arg(VALUE tickets)
   return c_tickets;
 }
 
+static int
+check_min_tickets_arg(VALUE min_tickets)
```

Comment:
Should we have a default minimum? I'm not sure if it would ever make sense to have less than, say, 3 tickets? What are you planning on configuring this to for our use-case?